timeseries/src/autogluon/timeseries/models/chronos/pipeline/chronos.py

```diff
@@ -529,6 +529,16 @@ def from_pretrained(cls, *args, **kwargs):
         """
         kwargs = kwargs.copy()
 
+        # Remove all covariate-related kwargs because Chronos does not support fine-tuning with covariates
+        dynamic_dims = kwargs.pop("dynamic_dims", 0)
+        past_dynamic_dims = kwargs.pop("past_dynamic_dims", 0)
+        static_dims = kwargs.pop("static_dims", 0)
+        static_cardinalities = kwargs.pop("static_cardinalities", [])
+        dynamic_cardinalities = kwargs.pop("dynamic_cardinalities", [])
+        past_dynamic_cardinalities = kwargs.pop("past_dynamic_cardinalities", [])
```

Comment:
Do we need to set these to variables if they are just ignored afterwards? Maybe some filtering would be simpler:
```python
ignored_kwargs = ["dynamic_dims", "past_dynamic_dims", ...]
kwargs = {k: v for k, v in kwargs.items() if k not in ignored_kwargs}
```
Also surprising that ruff does not complain about unused variables.